### DISCUSSION TITLE
PNP Modern Search Results 4.# - Difference between Microsoft OOB search results

### DISCUSSION THREAD

**Discussion by renate115** [microsoft-search/pnp-modern-search#1404] [Q&A] [answered]
The search results returned are different that the ones that are returned from the Microsoft OOB search results. I have several customers who love the pnp modern search functionality, but the one thing that I can't duplicate is the same results that come back from the OOB search. I have it configured to use the standard Local SharePoint using the SharePoint Search data source. 
I was told that the results will be different, but there must be information on what is being returned from the OOB and how we can obtain same set of results. 

Thank you in advance

**Comment by wobba** [comment, marked as answer]
This is "by design" as OOB search uses a different ranking mechanism not available to the SharePoint API provider. If you use Graph you will see ranking more equal to OOB.
